src/providers/slot-selector/slot-selector.ts

```diff
@@ -56,14 +56,21 @@ export class SlotSelectorProvider {
     return gotValue;
   }
 
+  public isTestSlot = (slot:Slot) => {
```

Comment:
this can just be a one-liner `public isTestSlot = (slot:Slot) => has(slot, 'vehicleSlotType');`